uber/site_sections/jobs.py

```diff
@@ -170,10 +171,7 @@ def staffers_by_job(self, session, id, message=''):
         return {
             'job':       job,
             'message':   message,
-            'attendees': session.query(Attendee.id, Attendee.full_name)
-                                .filter_by(staffing=True, **({'trusted': True} if job.restricted else {}))
-                                .filter(Attendee.assigned_depts.contains(str(job.location)))
-                                .order_by(Attendee.full_name).all()
```

Comment:
See my note about about making this a property on the ``Job`` class.  Not a huge deal, but if it is a property on that class then you don't need to pass this as a parameter and can instead just say ``job.staffer_opts`` or whatever.